BossMod/Framework/Plugin.cs

```diff
@@ -161,6 +162,38 @@ private void OnCommand(string cmd, string args)
                 case "r":
                     _wndReplay.SetVisible(!_wndReplay.IsOpen);
                     break;
+                case "aion":
```

Comment:
Why is that needed, considering there's already a 'cfg' command for modifying arbitrary options?